draft-irtf-cfrg-vdaf.md

```diff
@@ -260,8 +260,8 @@ informative:
   TestVectors:
     title: "Test vectors for Prio3 and Poplar1"
     target: "https://github.com/cfrg/draft-irtf-cfrg-vdaf/tree/main/test_vec/vdaf"
```

Comment:
The style guide also says to use the URL of the main page of the repository. Maybe we should do that and mention the file path in the appendix where we refer to this?

Reply:
Done.